digital_output.go

```diff
@@ -14,7 +14,7 @@ const (
 	// DoFalseValue digital output false value to write
 	DoFalseValue = "0\n"
 	// DoFolderRegex regular expression used for finding folders which contain digital output
-	DoFolderRegex = "do_[0-9]_[0-9]{2}"
+	DoFolderRegex = "[dr]o_[0-9]_[0-9]{2}"
```

Comment:
I've also noticed this one was really missing :+1:

Reply:
I have addressed this in the last push, in 59af0eb75ced1b33838e9eb53130ed785cc019ac